nnstreamer.ini.in

```diff
@@ -26,4 +26,9 @@ enable_use_gpu=@TORCH_USE_GPU@
 [nnfw-runtime]
 prioritize_tflite_ext=@NNFW_RUNTIME_PRIORITIZE@
```

Comment:
TODO
prioritize_tflite_ex should be removed after setting the fw priority.

Reply:
Okay, I'll integrate it later.